lib/go/test/topshot_test.go

```diff
@@ -398,7 +398,7 @@ func TestMintNFTs(t *testing.T) {
 		assert.Equal(t, cadence.NewBool(true), result)
 
 		result = executeScriptAndCheck(t, b, templates.GenerateGetCollectionIDsScript(env), [][]byte{jsoncdc.MustEncode(cadence.Address(topshotAddr))})
-		idsArray := cadence.NewArray([]cadence.Value{cadence.NewUInt64(1), cadence.NewUInt64(2), cadence.NewUInt64(3), cadence.NewUInt64(4), cadence.NewUInt64(5), cadence.NewUInt64(6)})
+		idsArray := cadence.NewArray([]cadence.Value{cadence.NewUInt64(6), cadence.NewUInt64(4), cadence.NewUInt64(2), cadence.NewUInt64(1), cadence.NewUInt64(5), cadence.NewUInt64(3)})
```

Comment:
@joshuahannan result is now returning as [6, 4, 2, 1, 5, 3] each run. Explored assert.Contains but that was not playing well with cadence.value. Any suggestions appreciated, or perhaps this is the only way to do it.

Reply:
I think the latest version of Cadence had a change to how arrays are ordered, or something like that... I think this is fine.